server/elasticsearch.ts

```diff
@@ -74,6 +74,48 @@ export default class Elasticsearch {
     }));
   }
 
+  /**
+   * Gets values from a specific field from all indices (except if excluded) for one study.
+   * @param id the identifier of the study.
+   * @param field the name of the field to return values from.
+   * @param excludeIndex optionally exclude results from an index.
+   */
+  async getStudyFieldAllIndices(id: string, field: keyof CMMStudy, excludeIndex?: string) {
```

Comment:
This is a generic method of getting contents of fields back from the index but since `field` is only `relatedPublications` and there is no other usage there is no use having the flexibility. If we need the flexibility in the future we can implement it then.